### DISCUSSION TITLE
Branch Name Pattern Not Explained

### DISCUSSION THREAD

**Discussion by theduke** [community/community#37861] [Repositories] [answered]
The "Branch name pattern" setting in repositories is not explained in the UI.

Is it a glob? Is it a regex?

There should be a link to documentation in the UI.

**Comment by airtower-luna** [comment, marked as answer]
It's a weird thing in between, but a simple `*` glob should work. :sweat_smile: The documentation describes it here: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule#about-branch-protection-rules

**Reply by theduke** [reply]
I actually know about the documentation, this is more just a suggestion to add a link to it in the UI.
